peering/tables/columns.py

```diff
@@ -27,3 +27,14 @@ def __init__(self, *args, **kwargs):
             transform=lambda p: p.get_type_html(display_name=True),
             **kwargs,
         )
+
+
+class CommunityColumn(tables.ManyToManyColumn):
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            default=mark_safe('<span class="text-muted">&mdash;</span>'),
+            separator=" ",
+            transform=lambda p: p.get_type_html(display_name=True),
+            **kwargs,
+        )
```

Comment:
Not required and can be figured out later, but since this is the same as the `RoutingPolicyColumn` maybe we can make it one class and rename it. Not sure about the naming though.

Reply:
Removing this class broke things (As I included it somewhere)